wolvic/browser/vr/wvr_device.cc

```diff
@@ -49,6 +49,7 @@ WvrDevice::WvrDevice()
   }
 
   SetSupportedFeatures(device_features);
+  SetArBlendModeSupported(true);
```

Comment:
Why is this "true" always ?

Reply:
Because we need to notify the web engine that wolvic as a provider is AR capable. Ideally I agree that we should wait until blend modes are received and then check if there is any compatible with AR sessions. But even in that case you could never be 100% sure, for example Meta devices do not support any AR blend modes but provide an OpenXR extension for passthrough